desktop/funnelcake136/distribution/distribution.ini

```diff
@@ -0,0 +1,11 @@
+# Partner Distribution Configuration File
+
+[Global]
+id=mozilla136
+version=1.0
+about=Funnelcake Grammarly Test
+
+[Preferences]
+app.partner.mozilla136="mozilla136"
+startup.homepage_welcome_url="https://www.mozilla.org/%LOCALE%/%APP%/%VERSION%/firstrun/?f=136"
```

Comment:
Is this finalized or we'll need to come back and confirm ?

Reply:
I think we'll need to come back and confirm.

There's also a problem that the grammarly WebExtension shows a firstrun page in front of ours.

I mainly wanted to get something built so the team could take a look.

Reply:
Ok, fair enough. You could either pick up the first 62 beta after this merges, or request a 61.0.2 respin of funnelcake.